### ISSUE TITLE
[FR] Add `llm logs rm`

### ISSUE THREAD

**Issue by NightMachinery** [simonw/llm#366]
`llm logs rm <cid>` will remove the conversation with the given ID from the database. This removal should completely remove the data from the database, and not just add a delete flag or sth.

`llm logs rm --all` should remove all the logs.

`llm logs rm --hours 24` should remove the logs of the previous 24 hours. `--days` etc.

**Comment by justinmayer** [comment]
I agree there should be a built-in way to remove logged queries and responses. Until that capability is implemented, the only option seems to be to remove the database manually, which on MacOS Sequoia I did via:

    trash "~/Library/Application Support/io.datasette.llm/logs.db"

**Comment by NightMachinery** [comment]
Is this still not implemented?

**Comment by vwkd** [comment]
+1 Additionally, removing individual request/response pairs from a conversation, for example when you accidentally continued the wrong one.
